src/i18n/pt-BR/ui.ts

```diff
@@ -1,6 +1,15 @@
 import { UIDictionary } from '../translation-checkers';
 
 export default UIDictionary({
+	'a11y.skipLink': 'Pular para o Conteúdo',
+	'navbar.a11yTitle': 'Navegação Superior',
+	'site.title': 'Documentação do Astro',
+	'site.description': 'Construa websites mais rápidos com menos Javascript no lado do cliente.',
```

Comment:
What if we add a conector like this? Maybe it would feel more natural:
```suggestion
	'site.description': 'Construa websites mais rápidos e com menos Javascript no lado do cliente.',
```

Reply:
Maybe we should change the wording a bit? I want to be clear that the website is faster because we're sending less Javascript to the client-side, if we add `e` it looks a bit like it's two separate things and not the reason why it is faster.

Reply:
Nice, I agree!